src/routes/getLockCategories.ts

```diff
@@ -13,11 +13,15 @@ export async function getLockCategories(req: Request, res: Response): Promise<Re
 
     try {
         // Get existing lock categories markers
-        const lockedCategories = await db.prepare('all', 'SELECT "category" from "lockCategories" where "videoID" = ?', [videoID]) as {category: Category}[];
-        if (lockedCategories.length === 0 || !lockedCategories[0]) return res.sendStatus(404);
-        // map to array in JS becaues of SQL incompatibilities
-        const categories = Object.values(lockedCategories).map((entry) => entry.category);
+        const row = await db.prepare('all', 'SELECT "category", "reason" from "lockCategories" where "videoID" = ?', [videoID]) as {category: Category, reason: string}[];
+        // map categories to array in JS becaues of SQL incompatibilities
+        const categories = row.map(item => item.category);
+        if (categories.length === 0 || !categories[0]) return res.sendStatus(404);
+        // Get longest lock reason
+        const reason = row.map(item => item.reason)
```

Comment:
Should we get the latest one instead?

Reply:
I assumed that the longest one would have the most information, while the latest might just be relying on an old warning message

Reply:
https://discord.com/channels/603643120093233162/603643299961503761/863870164108574731

Reply:
Got it, will use the latest reason